Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM alpine:3.19.1 as builder
+FROM alpine:3.20.0 as builder
```

Comment:
To reduce maintenance burden, you should be able to just go with 3.20 or even alpine:3

Reply:
Sure, not a good practice to show, but for this purpose, agree with you, let's do this. Fixed.